libcrux-ml-kem/src/arithmetic.rs

```diff
@@ -46,8 +45,8 @@ const BARRETT_MULTIPLIER: i64 = 20159;
 ///
 /// In particular, if `|value| < BARRETT_R`, then `|result| < FIELD_MODULUS`.
 
-#[cfg_attr(hax, hax_lib::requires((i64::from(value) > -BARRETT_R && i64::from(value) < BARRETT_R)))]
-#[cfg_attr(hax, hax_lib::ensures(|result| result > -FIELD_MODULUS && result < FIELD_MODULUS))]
+#[cfg_attr(hax, hax_lib_macros::requires((i64::from(value) > -BARRETT_R && i64::from(value) < BARRETT_R)))]
```

Comment:
I don't know what is the right crate over in this branch: hax_lib or hax_lib_macros...

Reply:
Oh, it's both by now. But really, we should use `hax_lib` everywhere. It's re-exported now.